app/views/admin/adoption_applications/show.html.erb

```diff
@@ -0,0 +1,39 @@
+<h1>Admin Application Show Page</h1>
+<p>Application ID: <%= @adoption_app.id %></p>
+<p>Name: <%= @adoption_app.name %></p>
+<p>Street Address: <%= @adoption_app.street_address %></p>
+<p>City: <%= @adoption_app.city %></p>
+<p>State: <%= @adoption_app.state %></p>
+<p>Zip Code: <%= @adoption_app.zip_code %></p>
+<p>Why they want to adopt: <%= @adoption_app.description %></p>
+<p>Why they would be good owners: <%= @adoption_app.ownership_description %></p>
+<p>Status: <%= @adoption_app.status %></p>
+
+
+<h3>Pets applied for: </h3>
+<ul>
+<% @app_pets.each do |pet| %>
+    <div id="pet-<%= pet.id %>">
+    <li><%= link_to "#{pet.name}", "/pets/#{pet.id}" %></li>
+    <ul>
+        <li>Breed: <%= pet.breed %></li>
+        <li>Age: <%= pet.age %></li>
+        <li>Adoptable: <%= pet.adoptable %></li>
+    </ul>   
+    <% if AdoptionApplicationPet.where(pet_id: pet.id,adoption_application_id: @adoption_app.id).first.status == "Approved" %>
```

Comment:
`.where` is an AR method so we don't want to call it in the view per MVC conventions. Instead, you could create a model method in the Pet model that grabs the corresponding AdoptionApplicationPet with a given app ID. 

Another option would be to, on line 15, instead of iterating over an application's pets, you could iterate over an application's application_pets (i.e. iterate over the join objects) and then you could get the pet information by doing `adoption_application_pet.pet.breed` since each joins record belongs to the pet. And your if statement would then be simplified to `if adoption_application_pet.status == "approved"`